src/main/kotlin/org/malachite/estella/services/HrPartnerService.kt

```diff
@@ -17,6 +17,8 @@ class HrPartnerService(
 
     fun addHrPartner(hrPartner: HrPartner): HrPartner = hrPartnerRepository.save(hrPartner)
 
+    fun getHrPartnerByMail(mail: String?): HrPartner? = getHrPartners().find { it.user.mail == mail }
```

Comment:
Nie wiem, czy nie powinniśmy tego jakoś lepiej robić. W sensie teraz mamy takie długie łańcuszki tych parametrów - jedna zmiana w modelu i pół aplikacji trzeba zmieniac. Nie mówię, że to zmiana na teraz, ale nie wiem, czy odpowiednie serwisy się tym nie powinny zajmować, albo co. Chyba że jest to specyficzne dla hibernate'a

Reply:
Czyli chciałbyś dorobić funkcje typu ?
```kotlin
fun HrPartner.getMail() = user.mail
```

Reply:
No na przykład. W sensie to chyba jeszcze Apohllo mówił na wykładach z obiektowego, że takie łańcuszki są potworne.

Reply:
I tak i nie bo z drugiej strony będziesz miał masę getów które będą jednolinijkowcami